### ISSUE TITLE
Need help crafting pipelines to deploy to six single-tenant apps in the same AWS account

### ISSUE THREAD

**Issue by benmappen** [Armory/armory#22]
I have six installations of the same codebase (each in a different VPC) in a single AWS account. I'd like Armory's advice on the best way to deploy to all of them. Should I create a single pipeline that deploys to each in parallel (which requires stages to spin down workers before new ones spin up) or create a single pipeline per install?

![screen shot 2018-02-07 at 2 47 11 pm](https://user-images.githubusercontent.com/140719/36002079-b4b9bfbc-0cdd-11e8-9938-25e9d34595c9.png)

**Comment by benmappen** [comment]
Ryan comment: 

i realized that if i use 6 different pipelines, ill create 6 different AMI's with 6 different bakes.

Screenshot in description of what it would look like if i have just 2 of the installations. So i was thinking maybe one pipeline can build and then tag that AMI, then have 6 other pipelines that find the baked and tagged image?

**Comment by imosquera** [comment]
> So i was thinking maybe one pipeline can build and then tag that AMI, then have 6 other pipelines that find the baked and tagged image?

Yes.  That is exactly what you should do.  Having independent pipelines will allow you to run them independently from another.   But you could also group them together in one master pipeline so you could run them all together.

**Comment by rtloeffler** [comment]
@imosquera 
How would i pickup a tagged AMI from another pipeline?

**Comment by imosquera** [comment]
Add a "Find Images By Tag" stage and use the package name.
